packages/orchestration/src/facade.js

```diff
@@ -38,7 +39,7 @@ const makeLocalChainFacade = (
     },
 
     async makeAccount() {
-      const lcaP = E(localchain).makeAccount();
+      const lcaP = E.when(E(localchain).makeAccount());
```

Comment:
It's confusing to me that `E` has to wrap itself like this. Consider using V until we have the migration plan documented.
```suggestion
      const lcaP = V.when(E(localchain).makeAccount());
```